packages/react-magma-docs/gatsby-ssr.js

```diff
@@ -23,6 +23,17 @@ const StyledSkipLink = styled(SkipLink)`
   }
 `;
 
+const MainContainer = styled.div`
```

Comment:
Since `MainContainer` is used in multiple spots should it be pulled out into it's own file and then we can import it in both places?

Reply:
Yes, yes it definitely should be :). I will make that change